python/tk_desktop/desktop_window.py

```diff
@@ -722,6 +725,13 @@ def __launch_app_proxy_for_project(self, project, pipeline_configuration_id=None
             else:
                 raise RuntimeError("unknown platform: %s" % sys.platform)
 
+            if config_path is None:
+                raise RuntimeError("No path set for %s on the Pipeline "
+                                   "Configuration id %d. Please make sure that "
+                                   "it is set on your Shotgun website." %
```

Comment:
Should we also mention install_location.yml. Sometimes people edit those things manually.